src/hooks/core/useCoreChatSystemHook.ts

```diff
@@ -201,13 +200,6 @@ const useCoreChatSystemHook = () => {
       websocketData?.originalMembers?.forEach(async (member) => {
         const userMember = UserSchema.fromMemberWebsocketObject(member, websocketData?.channel?.id);
         await userMember.saveOrUpdateIfExists(localDb);
-
-        const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
-          websocketData?.channel?.id,
-          websocketData?.message?.id,
-          member
-        );
-        await memberSchema.save(localDb);
       });
     } catch (e) {
       console.log('error on memberSchema');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation removes the creation and saving of `memberSchema` objects. If these objects are not required anymore, then this change is fine. However, if they are still needed in other parts of your application, this could introduce bugs. Please ensure that removing this does not affect any other part of your system.
```diff
-        websocketData?.originalMembers?.forEach(async (member) => {
-          const userMember = UserSchema.fromMemberWebsocketObject(member, websocketData?.channel?.id);
-          await userMember.saveOrUpdateIfExists(localDb);
-        });
+        websocketData?.originalMembers?.forEach(async (member) => {
+          const userMember = UserSchema.fromMemberWebsocketObject(member, websocketData?.channel?.id);
+          await userMember.saveOrUpdateIfExists(localDb);
+
+          const memberSchema = ChannelListMemberSchema.fromWebsocketObject(
+            websocketData?.channel?.id,
+            websocketData?.message?.id,
+            member
+          );
+          await memberSchema.save(localDb);
+        });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->